portality/templates/doaj/article.html

```diff
@@ -208,9 +208,17 @@ <h1>&lt; Article title missing &gt;</h1>
         <p>
             <strong>
                 Abstract
-                {% for url in bibjson.get_urls("fulltext") %}
-                    | <a href="{{url}}">Full Text</a>
-                {% endfor %}
+                {% if bibjson.get_urls("fulltext") %}
+                    {% for url in bibjson.get_urls("fulltext") %}
+                        | <a href="{{url}}">Full Text</a>
+                    {% endfor %}
+                {% else %}
+                    {% if doi %}
+                        {% set tendot = doi[doi.find('10.'):] %}
+                        {% set href = "https://doi.org/{0}".format(tendot) %}
+                        <a href="{{ href }}"> Full Text </a>
```

Comment:
This can be simplified - we don't need to set `tendot` here in the template because of the changes you made to the template filter in `app.py`. It can look a lot more like the doi link generation above.